scripts/build_workflow_release.sh

```diff
@@ -42,14 +41,17 @@ function deploy_options() {
   local base_options
   if [ -f ${wdl_dir}/${prefix}.options.json ]; then
     base_options=${prefix}.options.json
+    cp ${wdl_dir}/${base_options} ${target_dir}/${versioned_options}
   elif [ -f ${wdl_dir}/${prefix}.${env}.options.json ]; then
     base_options=${prefix}.${env}.options.json
+    cp ${wdl_dir}/${base_options} ${target_dir}/${versioned_options}
+  elif [ -f ${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json ]; then
+    cp ${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json ${target_dir}/${versioned_options}
   else
     echo >&2 Error: Options JSON not found at either ${prefix}.options.json or ${prefix}.${env}.options.json
     exit 1
   fi
 
-  cp ${wdl_dir}/${base_options} ${target_dir}/${versioned_options}
```

Comment:
If you set base_options for all tree paths, you can keep this here and remove the other instances above.

Reply:
I think I made this change work, if you want to take a second look.

Reply:
looks good!